aspnetcore/release-notes/aspnetcore-10.0.md

```diff
@@ -53,6 +53,9 @@ This section describes new features for OpenAPI.
 
 [!INCLUDE[](~/release-notes/aspnetcore-10/includes/webapiaotTemplateAddedOpenAPI.md)]
 
+[!INCLUDE[](~/release-notes/aspnetcore-10/includes/xml-comment-generator.md)]
```

Comment:
@tdykstra  always add this so we can review the build. I've been leaving it in, it's unlikely to cause a merge conflict. If a merge conflict is identified, you can always remove the include.